src/main/java/racing/model/Cars.java

```diff
@@ -0,0 +1,32 @@
+package racing.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+
+    public List<Car> carList = new ArrayList<>();
+    int playTime;
+
+    Decider decider = new Decider();
+
+    public void setCar(String[] carNames, int playTime) {
+        this.playTime = playTime;
+
+        for(String car : carNames) {
+            carList.add(new Car(car, playTime));
+        }
+    }
+
+    public List<Car> getCarList() {
+        return this.carList;
+    }
+
+    public void move() {
+        for(Car car : carList) {
```

Comment:
향상된 for문도 좋지만 java 8의 stream을 이용해 봅시다. forEach는 stream 안써도 걍 쓸 수 있어요. 함수형 프로그래밍에 익숙해지면 코드가 간결해지고 가독성이 올라갑니다
```suggestion
      carList.forEach(car -> decider.decideMove(car));
```